src/model.py

```diff
@@ -172,17 +174,19 @@ async def generate(self, request):
         self.ongoing_request_count += 1
         try:
             request_id = random_uuid()
-            prompt = pb_utils.get_input_tensor_by_name(request, "PROMPT").as_numpy()[0]
+            prompt = pb_utils.get_input_tensor_by_name(
+                request, "text_input"
+            ).as_numpy()[0]
             if isinstance(prompt, bytes):
                 prompt = prompt.decode("utf-8")
-            stream = pb_utils.get_input_tensor_by_name(request, "STREAM").as_numpy()[0]
+            stream = pb_utils.get_input_tensor_by_name(request, "stream").as_numpy()[0]
```

Comment:
one other change to be picked up - making stream an optional param:

https://github.com/triton-inference-server/tutorials/pull/58/files

Reply:
Its already included in the changes.

Reply:
https://github.com/triton-inference-server/vllm_backend/blob/40f38ff5c783624bf32027f8d69e77d187e028de/ci/qa_models/vllm_opt/config.pbtxt#L45

Reply:
@tanmayv25 but don't we need to check here if it is included?

```suggestion
            stream = pb_utils.get_input_tensor_by_name(request, "stream")
            if stream:
               stream = stream.as_numpy()[0]
            else:
               stream = False
```

Reply:
Good catch! I missed it.
Probably one more test case to include in our CI tests.
@pskiran1 ^^^